cmake/blas.cmake

```diff
@@ -0,0 +1,67 @@
+# *******************************************************************************
+# Copyright 2020 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+if(blas_cmake_included)
+    return()
+endif()
+set(blas_cmake_included true)
+include("cmake/options.cmake")
+
+if(DNNL_TARGET_ARCH STREQUAL "AARCH64")
```

Comment:
Just as a idea: would just using `BLA_VENDOR` work? The logic would slightly change, but I believe that this would result in fewer build options?

Reply:
Do you mean using `BLA_VENDOR` as a oneDNN build option, or adding a `DNNL_BLAS_VENDOR` option which just maps to `BLA_VENDOR`.
In the ArmPL case this would leave users to choose the OMP or serial version when they build, rather than taking the hint from `DNNL_CPU_RUNTIME` - I'm not sure if that's better or worse! I guess I don't object to such a change.

Reply:
I guess this is resolved as we now use `DNNL_BLAS_VENDOR`.